static_src/components/info_app_create.jsx

```diff
@@ -0,0 +1,65 @@
+
+import React from 'react';
+
+import style from 'cloudgov-style/css/cloudgov-style.css';
+import { config } from 'skin';
+
+import createStyler from '../util/create_styler';
+
+const propTypes = {
+  org: React.PropTypes.object,
+  space: React.PropTypes.object
+};
+
+const defaultProps = {
+};
+
+
+export default class SpaceContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.styler = createStyler(style);
+  }
+
+  render() {
+    const org = this.props.org || {};
+    const space = this.props.space || {};
+
+    return (
+      <div className={ this.styler('info', 'info-app_create') }>
+        <p>
+          Learn how to <a href={ config.docs.deploying_apps }>deploy a new app</a>.
```

Comment:
The app would crash, or the link would just be null. I think this is really a problem with the current skin config design, so I'd rather not fix it here. The skin config should make it so it's okay to have missing keys and get sane defaults or at least not crash.

If you wanted to fix it here, (and everywhere the config is used) it looks like this:
```javascript
import { config } from 'skin';

// For each key you're using:
const deployingApps = config.docs && config.docs.deploying_apps || "https://cloudfoundry.org";
```